languoids/tree/arti1236/toki1239/md.ini

```diff
@@ -6,6 +6,8 @@ iso639-3 = tok
 hid = tok
 countries = 
 	US
+macroareas = 
+	UNKNOWN
```

Comment:
@d97hah I had fixed (i.e. removed) these before. Maybe you need to fix your side of the machinery that keeps re-inserting?